File: best-slide.js

const bestSlide = (n) => {
    let result = 0;
    let counter = 1;
    for(let i=1; i<=n; i++){
        result += counter;
        for(let j=1; j<=i; j++){
            process.stdout.write(`${counter} `);
            counter++;
        }
        console.log("");
        
    }
    console.log("");
    return result;
}
console.log("Answer: ", bestSlide(4));
console.log("Answer2: ", bestSlide(10));

File: best-slide.py
def bestSlide(n):
    result = 0
    counter = 1
    for i in range(1, n+1):
        result += counter
        for j in range(1, i+1):
            print(counter, end=" ")
            counter += 1
        print("")
    
    print("")
    return result

print("Answer: ", bestSlide(4))
print("Answer2: ", bestSlide(10))